examples/class_increment_semantic_segmentation/lifelong_learning_bench/testalgorithms/erfnet/ERFNet/mypath.py

```diff
@@ -0,0 +1,20 @@
+class Path(object):
```

Comment:
Hardcoded path, it is suggested to return these path without the personal path like 'robo' 'm0063' etc..